stable/node-problem-detector/values.yaml

```diff
@@ -2,6 +2,7 @@ settings:
   log_monitors:
     - /config/kernel-monitor.json
     - /config/docker-monitor.json
+  custom_plugin_monitors: {}
```

Comment:
Here you are saying it's a map but I think the iteration you have used in `command` is expecting a list. Correct?

Reply:
Fixed.